tensorflow/compiler/xla/service/gpu/ir_emitter_unnested.cc

```diff
@@ -2405,7 +2405,14 @@ Status IrEmitterUnnested::EmitTargetElementLoop(
 
   bool few_waves = false;
   auto is_good_for_few_waves = [](const HloInstruction* instr) {
-	return instr->opcode() != HloOpcode::kReduceWindow;
+    if (instr->opcode() == HloOpcode::kReduceWindow) {
```

Comment:
How about this: instead of providing a blacklist of instructions not to apply few_waves, have a white list. Maybe the list can be `return instr->IsElementwise() || instr->opcode() == HloOpcode::kReduce || ...`. We make it just enough to capture GELU and some fusions that has GELU.

Having a whitelist is also safe for the production, as few_waves seems to be unstable across all instructions.